components/esp_lvgl_port/Kconfig

```diff
@@ -0,0 +1,15 @@
+menu "ESP LVGL PORT"
+
+    config ESP_LVGL_PORT_TASK_CORE_EN
+        bool "Enable pin LVGL task to selected core"
```

Comment:
Any reason for Kconfig vs part of `ESP_LVGL_PORT_INIT_CONFIG`? Also this option should be hidden / disabled when `FREERTOS_UNICORE` is enabled

Reply:
I thought about it, if I should put it into init configuration structure and I think, that this is better solution for this selection. It cannot be selected in runtime. I will add FREERTOS_UNICORE depend to Kconfig.

Reply:
I thought, that the configuration structure is only for realtime settings and this seems to be ok in compilation settings only. I didn't know, that we have it in another place in configuration structure. Should I change it?

Reply:
If the option can be both in menuconfig or run-time config, run-time config is preferred. (Same with task's stack size, and task's priority ...).

So yes, please change it unless there is a reason to have it in menuconfig :)

Reply:
Ok, changed.